test/extended/openstack/volumes.go

```diff
@@ -280,6 +352,17 @@ func getMonitoringPvcs(ctx context.Context, dc dynamic.Interface) ([]objx.Map, e
 	return objects(objx.Map(obj.UnstructuredContent()).Get("items")), nil
 }
 
+// return list of PVCs defined in a given namespace
+func GetPvcsFromNamespace(ctx context.Context, dc dynamic.Interface, namespace string) ([]objx.Map, error) {
```

Comment:
Any chance we could capitalise PVC?

```suggestion
func GetPVCsFromNamespace(ctx context.Context, dc dynamic.Interface, namespace string) ([]objx.Map, error) {
```

Reply:
Sure! Done.